i18n/locales/en.ts

```diff
@@ -487,19 +487,29 @@ const translations = {
       paragraph_four: 'Him again?',
     },
     resources: {
-      hashing: {
+      hashing_two: {
         hash_functions_heading: 'Hash Functions',
         hash_functions_paragraph:
           'A hash function is any function that can be used to map data of arbitrary size to fixed-size values. The values returned by a hash function are called hash values, hash codes, digests, or simply hashes. The values are usually used to index a fixed-size table called a hash table. Use of a hash function to index a hash table is called hashing or scatter storage addressing.',
+        tip: 'Just keep typing! Because of the  SHA256 algorithm any new input will result in a completely random input even if those inputs are very similar',
+        spoiler: 'Try something with 8 characters.',
+      },
+      hashing_four: {
+        power_of_random_heading: 'The Power of Random',
+        power_of_random_paragraph:
+          "Like a perfectly random scenario in probability the sha256 hash function is completely random for each addition of new data adding a single letter to 'War and Peace' will genereate a completely random hash to one without it. Since each new character recalculates the hash to be completely random and with 16 possible characters in a hex decimal system, 0-9 and a-f how likely is it to find a hash starting with '0'?",
```

Comment:
```suggestion
          "Like a perfectly random scenario, probability-wise the SHA-256 hash function is completely random for each addition of new data. If you hashed the novel 'War and Peace', then added just one extra letter to the original text, it would result in a completely different hash. Each new character changes the hash in a completely random manner. Considering that there are 16 possible characters in a hexadecimal system (0-9 and a-f), how likely is it to find a hash starting with '0'?",
```